app/controllers/flyover_comments/votes_controller.rb

```diff
@@ -0,0 +1,67 @@
+require_dependency "flyover_comments/application_controller"
+require_dependency "flyover_comments/authorization"
+
+module FlyoverComments
+  class VotesController < ApplicationController
+    include FlyoverComments::Authorization
+
+    respond_to :json, only: [:create, :destroy]
+
+    before_action :load_vote, only: [:update, :destroy]
+
+    def create
+      @comment = FlyoverComments::Comment.find(params[:comment_id])
+      authorize_flyover_vote_create!
+      @vote = @comment.votes.new(value: params[:value])
+      @vote._user = send(FlyoverComments.current_user_method.to_sym)
+
+      if @vote.save
+        @comment.recalculate_vote_counts!
```

Comment:
Probably doesn't need to be in the save. Also there are some annoyances with updating the votes count for each of these if we'd increment since we have to know what the old value of that vote is. So I just went with recalculate on each for now.